Templates/Common-Backend-Scripts/dbbBuild.sh

```diff
@@ -210,6 +209,15 @@ if [ $rc -eq 0 ]; then
     source $buildUtilities
   fi
 
+  # Read and import utilities
+  if [ ! -f "${fetchBuildDependenciesUtilities}" ]; then
+    rc=8
+    ERRMSG=$PGM": [ERROR] DBB-Build internal utilities (${fetchBuildDependenciesUtilities}) was not found. rc="$rc
+    echo $ERRMSG
+  else
+    source $fetchBuildDependenciesUtilities
```

Comment:
I think sourcing is not necessary if we just call the script later.

Reply:
Agreed, fixed that